packages/backend/src/rates/util.test.ts

```diff
@@ -6,10 +6,10 @@ describe('Rates util', () => {
       test.each`
         exchangeRate | sourceAmount | assetScale | expectedResult                                    | description
         ${1.5}       | ${100n}      | ${9}       | ${{ amount: 150n, scaledExchangeRate: 1.5 }}      | ${'exchange rate above 1'}
-        ${1.1602}    | ${12345n}    | ${2}       | ${{ amount: 14323n, scaledExchangeRate: 1.1602 }} | ${'exchange rate above 1 with rounding up'}
+        ${1.1602}    | ${12345n}    | ${2}       | ${{ amount: 14322n, scaledExchangeRate: 1.1602 }} | ${'exchange rate above 1 with rounding up'}
```

Comment:
small thing, but we should probably change the descriptions here if we go with `Math.floor`. the "with rounding up" and "with rounding down" seem to clarify how the `Math.round` was behaving.